packages/indexer-client/src/types/VertexTx.ts

```diff
@@ -26,8 +26,20 @@ export interface VertexLiquidateSubaccountTx {
   };
 }
 
+export interface EndpointWithdrawCollateralTx {
```

Comment:
Do we need this extra type? In places where we reference this, we could potentially just do `VertexWithdrawCollateralTx['withdraw_collateral']`